lib/matplotlib/cbook.py

```diff
@@ -2697,40 +2715,49 @@ class _StringFuncParser(object):
     _funcs['linear'] = _FuncInfo(lambda x: x,
                                  lambda x: x,
                                  True)
-    _funcs['quadratic'] = _FuncInfo(lambda x: x**2,
-                                    lambda x: x**(1. / 2),
+    _funcs['quadratic'] = _FuncInfo(np.square,
+                                    np.sqrt,
                                     True)
     _funcs['cubic'] = _FuncInfo(lambda x: x**3,
```

Comment:
I agree that FuncNorm should support that (even though it is not as useful as it may seem, imagine a case where the variable is the second argument and not the first one). Anyway we can discuss that in the next PR.

IMO doing that here is a mistake, as it goes against what we are trying to achieve, which is to provide the simplest possible way to specify a simple function. For more complex cases, the user will always be able to use FuncNorm with callables directly, or with a new kwargs option based on your suggestion.